Client/Base/frontend/components/basket/basket.component.scss

```diff
@@ -1,35 +1,39 @@
+// Global Imports
 @import "globals";
 
+// Component Styles
 :host {
-	background-color: var(--color-background);
-	width: 430px;
-	height: 100%;
-
-	h5 {
-		font-size: 25px;
-		font-weight: 500;
-		color: var(--color-text);
-	}
-
-	h4 {
-		font-size: 30px;
-		font-weight: 500;
-		color: var(--color-text);
-		margin: 0;
-	}
+    background-color: var(--color-background);
+    width: 430px;
+    height: 100%;
+
+    h5 {
+        font-size: 25px;
+        font-weight: 500;
+        color: var(--color-text);
+    }
+
+    h4 {
+        font-size: 30px;
+        font-weight: 500;
+        color: var(--color-text);
+        margin: 0;
+    }
 }
 
 .color {
-	color: var(--color-text);
+    color: var(--color-text);
 }
 
 .pointer {
     cursor: pointer;
 }
 
+// Styles for Whole Basket Component
```

Comment:
do not add so much comments, from class name it is already clear what we deal with